--- legesmotus-0.4.0/common.mk~	2010-09-03 21:22:37.000000000 +0200
+++ legesmotus-0.4.0/common.mk	2010-09-08 08:42:15.599068157 +0200
@@ -169,21 +169,18 @@
 
 INCLUDES = -I$(BASEDIR)
 
-CFLAGS += -Wall
 CXXFLAGS += -Wnon-virtual-dtor
 
 ifeq ($(DEBUG),1)
  RELEASE = debug
- CFLAGS += -g -O0 -DLM_DEBUG
+ CFLAGS += -g -DLM_DEBUG
 else
  RELEASE = release
  # Take O-flag from CFLAGS if we already have it set
- ifneq ($(filter -O%,$(CFLAGS)),)
-  CFLAGS += -O2
- endif
 endif
 
 LDFLAGS += $(CFLAGS)
+LDFLAGS += $(OLDFLAGS)
 CFLAGS += $(FLAGS_GL) $(INCLUDES)
 
 # Windows needs Winsock2 for BSD sockets
@@ -194,6 +191,7 @@
 CLIENTFLAGS = $(FLAGS_SDL)
 CLIENTLIBS = $(LIBS_GL) $(LIBS) $(LIBS_SDL)
 
+CFLAGS += $(OPTFLAGS)
 CXXFLAGS += $(CFLAGS)
 
 # Out-of-tree build madness
