--- legesmotus-0.3.0/common.mak~	2010-04-25 00:40:51.000000000 +0200
+++ legesmotus-0.3.0/common.mak	2010-04-27 15:49:16.748823957 +0200
@@ -109,16 +109,10 @@
 
 INCLUDES = -I$(BASEDIR)
 
-CFLAGS += -Wall
 CXXFLAGS += -Wnon-virtual-dtor
 
-ifeq ($(DEBUG),1)
- CFLAGS += -g -O0
-else
- CFLAGS += -O2
-endif
-
 LDFLAGS += $(CFLAGS)
+LDFLAGS += $(OLDFLAGS)
 CFLAGS += $(FLAGS_GL) $(INCLUDES)
 ifeq ($(MACHINE),Windows)
  LIBS += -lwsock32
@@ -126,6 +120,7 @@
 CLIENTFLAGS = $(FLAGS_SDL)
 CLIENTLIBS = $(LIBS_GL) $(LIBS) $(LIBS_SDL)
 
+CFLAGS += $(OPTFLAGS)
 CXXFLAGS += $(CFLAGS) -fno-rtti
 
 %.o: %.rc
